cmd/detect.go

```diff
@@ -81,6 +81,23 @@ func runDetect(cmd *cobra.Command, args []string) {
 		log.Fatal().Err(err).Msg("")
 	}
 
+	gitleaksPath, err:= cmd.Flags().GetString("gitleaks-ignore-path")
```

Comment:
@pacorreia could we rename this variable to `gitleaksIgnorePath`?

Reply:
Done